### PR TITLE
Feature: Remove wraith add backstopjs

### PR DESCRIPTION
**Pull request by rigoucr**
- Verify if all the related configuration with wraith was totally deleted.
- Please take a look of the new backstopJS config
- Generate a new project
- Verify is the variables in the template files was replaced correctly.
- Setup the new project (vagrant up and all the thinks, note: **don't forget the npm install** )
- Once ready your site find a similar site or create one and then configure the backstopjs (`backstop.json`) file.
- Finally in your root project directory execute: `npm run backstop` and check if backstop is generating the bitmaps and respective test.

### CODE REVIEW

File: generators/app/templates/7/_wercker.yml [hunk 1]
```diff
@@ -88,10 +78,6 @@ build:
             drush @<%= appName %>.<%= appName %>.dev ups | grep -qi "update available" && (echo 'Updates available FAIL' && exit 1) || (echo 'No updates available PASS' && exit 0)
             # Behat tests.
             ./.vendor/bin/behat -p wercker
-            # Visual regression tests.
-            # cd wraith && wraith capture capture
-            # Store wraith shots in wercker artifacts folder.
-            # cp -r shots $WERCKER_REPORT_ARTIFACTS_DIR/
```
Comment (kporras07) [inline generators/app/templates/7/_wercker.yml:94]: We're not testing backstop in wercker. Is that intended?
I'd create a new step for the backstop tests

**Comment by kporras07** [comment]
Overall code looks great!
I left a comment and then we can do functional review.

File: generators/app/templates/7/_wercker.yml (hunk 1 above)
Comment (rigoucr) [reply generators/app/templates/7/_wercker.yml:94]: @kporras07 the idea was just implement Backstop to locally use only and in the future as new feature, integrate it in the automated tests (circleCI)

**Comment by kporras07** [comment]
I fixed merge conflicts. This works ok. 
Please merge when travis passes. :shipit:

**Pull request merged**